src/routes/safe/components/Balances/SendModal/screens/ContractInteraction/utils/index.ts

```diff
@@ -73,7 +73,16 @@ export const formMutators = {
   },
 }
 
-export const handleSubmitError = (error, values) => {
+export const isAddress = (type: string): boolean => type.indexOf('address') === 0
+export const isBoolean = (type: string): boolean => type.indexOf('bool') === 0
+export const isString = (type: string): boolean => type.indexOf('string') === 0
+export const isUint = (type: string): boolean => type.indexOf('uint') === 0
+export const isInt = (type: string): boolean => type.indexOf('int') === 0
+export const isByte = (type: string): boolean => type.indexOf('byte') === 0
+
+export const isArrayParameter = (parameter: string): boolean => /(\[\d*])+$/.test(parameter)
```

Comment:
would probably escape the `]` for completeness, but should not be required at all.